expressjs/db/migrations/20200526095904_rebuild.js

```diff
@@ -0,0 +1,112 @@
+const { onUpdateTrigger } = require('../../knexfile');
+
+function dropTables(knex) {
+  let dropQuery = `
+    DROP TABLE if exists publications;
+    DROP TABLE if exists trails;
+    DROP TABLE if exists users;
+    DROP TABLE if exists cases;
+    DROP TYPE if exists state_type;
+    DROP TABLE if exists organizations;
+  `;
+  return knex.raw(dropQuery);
+}
+
+function buildOrganization(knex) {
+  return knex.schema.createTable('organizations', function (table) {
+    table.uuid('id').notNull().primary();
+    table.string('authority_name')
+    table.timestamp('updated_at').defaultTo(knex.fn.now())
+    table.timestamp('created_at').defaultTo(knex.fn.now())
+  }).then(() => knex.raw(onUpdateTrigger('organizations')));
+}
+
+function buildSettings(knex) {
+  return knex.schema.createTable('settings', function (table) {
+    table.uuid('id').notNull().primary()
+    table.uuid('organization_id').notNull().references('organizations.id').onDelete('CASCADE');
+    table.string('info_website_url')
+    table.string('reference_website_url')
+    table.string('api_endpoint_url')
+    table.json('region_coordinates')
+    table.integer('notification_threshold_percent').defaultTo(66);
+    table.integer('notification_threshold_count').defaultTo(6);
+    table.integer('chunking_in_seconds').defaultTo(43200);
+    table.timestamp('updated_at').defaultTo(knex.fn.now())
+    table.timestamp('created_at').defaultTo(knex.fn.now())
```

Comment:
We should have a field on this table that represents the number of days to retain records. Probably just a column named `numberOfDaysToRetainRecords` that stores an `Int`.